tests/unit/backend/layers/business/test_business.py

```diff
@@ -0,0 +1,802 @@
+        "test_tissue",
+        "test_assay",
+        "test_disease", 
+        "test_sex",
+        "test_self_reported_ethnicity",
+        "test_development_stage",
+        "test_cell_type",
+        10
+    )
+
+    test_user_name = "test_user_1"
+
+    user_info = UserInfo(
+        "test_user_1",
+        "fake token"
+    )
+
+    def setUp(self) -> None:
+        self.database_provider = DatabaseProviderInterface() # replace with DatabaseProviderMock()
+
+        # By default does nothing. Can be mocked by single test cases.
+        self.crossref_provider = CrossrefProviderInterface()
+        self.step_function_provider = StepFunctionProviderInterface()
+
+        self.business_logic = BusinessLogic(
+            database_provider=self.database_provider, 
+            crossref_provider=self.crossref_provider,
+            step_function_provider=self.step_function_provider,
+        )
+        return super().setUp()
+
+    def initialize_empty_private_collection(self, owner: str = test_user_name, metadata = sample_collection_metadata) -> CollectionVersion:
+        """
+        Initializes a private collection to be used for testing, with no datasets
+        """
+        version = self.database_provider.initialize_canonical_collection(
+            owner,
+            metadata,
+        )
+        return version
+
+    def initialize_private_collection(self, owner: str = test_user_name, metadata = sample_collection_metadata) -> CollectionVersion:
+        """
+        Initializes a private collection to be used for testing, with two datasets
+        """
+        version = self.initialize_empty_private_collection(owner, metadata)
+        for i in range(2):
+            dataset_version = self.database_provider.initialize_canonical_dataset(
+                version.version_id, 
+                dataset_metadata=self.sample_dataset_metadata
+            )
+            self.database_provider.add_dataset_to_collection_version_mapping(
+                version.version_id, 
+                dataset_version.version_id
+            )
+        return version
+        
+
+    def initialize_public_collection(self, owner: str = test_user_name, published_at: datetime = None, metadata = sample_collection_metadata) -> CollectionVersion:
+        """
+        Initializes a public collection to be used for testing, with a single dataset
+        """
+        version = self.initialize_private_collection(owner, metadata)
+        self.database_provider.finalize_collection_version(version.collection_id, version.version_id, published_at)
+        return version
+        
+
+
+class TestCreateCollection(BaseBusinessLogicTestCase):
+
+    def create_collection_ok(self):
+        """
+        A collection can be created using `create_collection`
+        """
+        cv = self.business_logic.create_collection(self.sample_collection_metadata, self.user_info)
+        cv_from_database = self.database_provider.get_collection_version(cv.version_id)
```

Comment:
start tests with `test_`. This helps separate tests from helper code.

Reply:
👍